apps-rendering/src/components/blockquote.tsx

```diff
@@ -20,24 +19,44 @@ const styles = (format: ArticleFormat): SerializedStyles => css`
 	font-style: italic;
 	position: relative;
 	margin: ${remSpace[4]} 0 ${remSpace[9]} 0;
-	padding: 0 ${remSpace[6]};
 
-	svg {
-		height: 62.4px;
-		margin-left: -9.2625px;
-		margin-bottom: -12px;
-		margin-top: 4.4px;
+	> svg {
```

Comment:
Mostly just nitpickity, feel free to ignore me, I don't think we use the `>` css operator much at all in this codebase

Reply:
Here, selecting the direct child is exactly what we want to do.

In some ways, if we select children, we should _only_ select direct children, otherwise you can have css cascade issues.